app/controllers/admin/statistics_announcement_publications_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::StatisticsAnnouncementPublicationsController < Admin::BaseController
+  before_action :find_statistics_announcement
+  layout "design_system"
+
+  def index
+    if params[:search].present?
+      filter
+    end
+  end
+
+  def connect
```

Comment:
Do you think we could just call this action `update`? that way we can just use an update resource rather than a custom route? 

```
resources :statistics_announcement_publications, as: "publication", path: "publication", only: %i[index update]
```

Reply:
Cool, I spent a bit of time playing around with this to get the button to style like a link, but wasn't happening quickly. So have decided to stick with current implementation.